populate repository for GLORIA data set pull (#1)
rossyndicate/Landsat_C2_SR_GLORIA: README.md

# Landsat_C2_SR_GLORIA

Code to pull, collate, and correct Landsat SR product at GLORIA and other sites as presented in Maciel et al. 2023.

This repository is covered by the MIT use license. We request that all downstream uses of this work be available to the public when possible.

## Code Origin

The code presented in this repository is an adaptation of the code in the [ROSSyndicate repository Landsat_C2_SRST](https://github.com/rossyndicate/Landsat_C2_SRST). This code was written by members of the ROSSyndicate at Colorado State University and other contributors. The alterations from the original workflow are as follows:

> 1.  only targets that pull buffered location data were retained (i.e. no center points or whole waterbodies)

Reviewer: ```suggestion
1.  only targets that pull buffered location data are utilized (i.e. no center points or whole waterbodies are pulled)
```

2.  the buffer method has been altered from a circular buffer to a square buffer to match the methods in Maciel, et al 2023.

3.  all thermal data and related columns have been removed for the purposes of this analysis.

> Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv".

Reviewer: Not quite sure what you mean in the first sentence - is there a typo? o_O

Author: lols. I'll fix this. This was a stray thought.

> Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv".

Author: ```suggestion
Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients were available. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv". The filtered dataset likely represents a fraction of the sites from the upstream pre-filtered datset.
```
